src/printer.js

```diff
@@ -1996,6 +2000,8 @@ function printFunctionParams(path, print, options) {
 
   return concat([
     "(",
+    comments.printDanglingComments(path, options, /* sameIndent */ true),
```

Comment:
You shouldn't have dangling comments if there are arguments, the comments are going to be attached to the elements. Have you seen a case when that's not happening?